lib/config.js

```diff
@@ -432,6 +432,8 @@ config.azureResourceURL = 'https://graph.windows.net';
 config.azureDestroySessionUrl =
   'https://login.microsoftonline.com/common/oauth2/logout?post_logout_redirect_uri=http://localhost:3000';
 
+config.enabledFeatures = ['course:manual-grading'];
```

Comment:
TODO: remove this before merging.

Should this be an array or an object? An object would allow us to override specific features while still respecting any remaining defaults. For instance, if this were the following:

```ts
config.features = {
  'course:manual-grading': true,
};
```

We could add the following JSON config and keep the above defaults:

```json
{
  "features": {
    "course:lti": true
  }
}
```

Reply:
@mwest1066 thoughts on this? Both on data format, and more generally if it makes sense to be able to globally override feature flags via the config. I'm thinking primarily about local development, where folks may be regularly nuking their DB via container restarts; it'd be useful to be able to maintain flags in the persistent config file.

Reply:
I agree it would be nice to be able to override specific flags from `config.json` for local dev. The "merge objects" syntax seems nice for this.

Reply:
This was changed in b918899.